src/sentry/api/endpoints/project_rule_details.py

```diff
@@ -124,6 +124,33 @@ def put(self, request: Request, project, rule) -> Response:
         if serializer.is_valid():
             data = serializer.validated_data
 
+            explicit_opt_out = request.data.get("optOutExplicit")
+            edit_opt_out = request.data.get("optOutEdit")
```

Comment:
why access the data directly from the request instead of adding it to the serializer?

Reply:
I did it this way because this is temporary data...ish? The intention is to get a list of rules that have fired x number of times that people haven't interacted with in y number of days, then schedule them for a disable migration that people can opt out of it they want. We may or may not repeat this in the future, but adding something that's likely temp the serializer felt weird.

Reply:
got it, makes sense. could be worth adding this comment to the code